test/external/ringtest/CMakeLists.txt

```diff
@@ -95,8 +96,8 @@ foreach(processor cpu gpu)
     NAME coreneuron_${processor}_mpi_threads
     REQUIRES coreneuron mpi python ${processor}
     PROCESSORS ${ringtest_mpi_ranks_times_threads}
-    COMMAND OMP_NUM_THREADS=${ringtest_num_threads} ${ringtest_special} -tstop 100 -coreneuron -nt
-            ${ringtest_num_datasets} ${gpu_arg})
+    ENVIRONMENT OMP_NUM_THREADS=${ringtest_num_threads}
```

Comment:
isn't there a problem because in the group the `OMP_NUM_THREADS` exists already and they overlap? Or did I miss something in the logic there?